client/src/App.js

```diff
@@ -22,7 +22,8 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Route exact path='/' component={MovieList}></Route>
+        <Route path='/movies/:id' component={Movie}></Route>
```

Comment:
"component" "c" not "C" . in the words Adele - "sometimes a lesson learn but sometime it hurts instead..."  Make it a lesson learnt lol